common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -137,7 +137,15 @@ class CapaFields(object):
             {"display_name": _("Finished"), "value": SHOWANSWER.FINISHED},
             {"display_name": _("Correct or Past Due"), "value": SHOWANSWER.CORRECT_OR_PAST_DUE},
             {"display_name": _("Past Due"), "value": SHOWANSWER.PAST_DUE},
-            {"display_name": _("Never"), "value": SHOWANSWER.NEVER}]
+            {"display_name": _("Never"), "value": SHOWANSWER.NEVER},
+            {"display_name": _("After # Attempts"), "value": SHOWANSWER.AFTER_ATTEMPTS}]
+    )
+    attempts_before_showanswer_button = Integer(
+        display_name=_("Show Answer - Attempts"),
```

Comment:
Similar to the pound sign, should 'Show Answer - Attempts' be 'Show Answer After Attempts' ?